README.md

```diff
@@ -11,31 +11,17 @@
 
 The Paramak python package allows rapid production of 3D CAD models of fusion
 reactors. The purpose of the Paramak is to provide geometry for parametric
-studies. It is possible to use the created geometry in engineering and
-neutronics studies as the STP or STL files produced can be automatically
-converted to DAGMC compatible neutronics models or meshed and used in
-finite element analysis codes.
+studies. The paramak can created geometry in standard CAD formats such as STP,
+STL. It can also create h5m files for 
+[DAGMC](https://svalinn.github.io/DAGMC/) based neutronics simulations.
 
-:point_right: [Documentation](https://paramak.readthedocs.io/en/main/)
+:point_right: [Documentation](https://paramak.readthedocs.io)
 
 :point_right: [Video presentation](https://www.youtube.com/embed/fXboew3U7rw)
 
 :point_right: [Publication](https://f1000research.com/articles/10-27/v1)
 
-# History
-
-The package was originally conceived by Jonathan Shimwell and based on the
-[FreeCAD Python API](https://wiki.freecadweb.org/FreeCAD_API). When 
-[CadQuery 2](https://github.com/CadQuery/cadquery) was released the project
-started to migrate the code base. Shortly after this migration the project
-became open-source and has flourished ever since. The project has grown largely
-due to two contributors in particular (John Billingsley and
-Remi Delaporte-Mathurin) and others have also helped, you can see all those who
-have helped the development in the 
-[Authors.md](https://github.com/ukaea/paramak/blob/main/AUTHORS.md) and in the 
-[GitHub contributions](https://github.com/ukaea/paramak/graphs/contributors).
-The code has been professionally reviewed by [PullRequest.com](https://www.pullrequest.com/) who
-produced a [report](https://github.com/ukaea/paramak/files/5704872/PULLREQUEST.Paramak.Project.Review.pdf) and inline [suggestions](https://github.com/ukaea/paramak/pull/639).
+:point_right: [Installation](https://paramak.readthedocs.io/en/main/#system-installation)
```

Comment:
perhaps this should go before the publication? or put the publication in the Citing section?

Reply:
I can move it before the publication